File: TransCoder_tokenized_test_set_functions/python/FIND_THE_MINIMUM_COST_TO_REACH_A_DESTINATION_WHERE_EVERY_STATION_IS_CONNECTED_IN_ONE_DIRECTION.py
def minCost ( cost ) :
    dist = [ 0 for i in range ( N ) ]
    for i in range ( N ) :
        dist [ i ] = INF
    dist [ 0 ] = 0
    for i in range ( N ) :
        for j in range ( i + 1, N ) :
            if ( dist [ j ] > dist [ i ] + cost [ i ] [ j ] ) :
                dist [ j ] = dist [ i ] + cost [ i ] [ j ]
    return dist [ N - 1 ]

File: TransCoder_tokenized_test_set_functions/java/FIND_THE_MINIMUM_COST_TO_REACH_A_DESTINATION_WHERE_EVERY_STATION_IS_CONNECTED_IN_ONE_DIRECTION.java
static int minCost ( int cost [ ] [ ] ) {
  int dist [ ] = new int [ N ];
  for ( int i = 0;
  i < N;
  i ++ ) dist [ i ] = INF;
  dist [ 0 ] = 0;
  for ( int i = 0;
  i < N;
  i ++ ) for ( int j = i + 1;
  j < N;
  j ++ ) if ( dist [ j ] > dist [ i ] + cost [ i ] [ j ] ) dist [ j ] = dist [ i ] + cost [ i ] [ j ];
  return dist [ N - 1 ];
}